src/juxt/site/alpha/handler.clj

```diff
@@ -207,7 +207,8 @@
                 {:ring.response/status 400}))))
 
       (with-open [in (:ring.request/body req)]
-        (let [body (.readNBytes in content-length)
```

Comment:
Doesn't this work? Are you trying to support earlier JDKs? What's the justification here?

Reply:
I describe the steps to reproduce the error in the first commit of this PR. Working my way through the dependencies for `ring-jetty-adapter 1.9.1`, the method `readNBytes` doesn't exist for https://javadoc.io/doc/org.eclipse.jetty/jetty-server/9.4.36.v20210114/org/eclipse/jetty/server/HttpInput.html

I believe I was on JDK8

Reply:
`readNBytes` is in `java.io.InputStream`, since Java 9. So the method doesn't exist in `HttpInput` but in its superclass. See https://docs.oracle.com/javase/9/docs/api/java/io/InputStream.html#readNBytes-byte:A-int-int-

Reply:
We had some issues before with streaming larger payloads, hence the use of `readNBytes` rather than reading the available bytes (which is only the bytes available in the receive buffer). Using content-length as the value of N seems to be OK. I suggest you try again with Java 9. Java 8 is out of support at this point any way.